tests/ppx/jbuild

```diff
@@ -0,0 +1,20 @@
+(executable
+ ((name suite)
+  (libraries (
+     alcotest
+     ppx_deriving
+     ppx_deriving_rpc
```

Comment:
This is not needed, `ppx_deriving_rpc` needs to be only in the `pps` section and the opam file